packages/protocol/contracts/governance/BlockchainParameters.sol

```diff
@@ -109,35 +109,31 @@ contract BlockchainParameters is Ownable, Initializable, UsingPrecompiles {
     require(window >= 3 && window <= 720, "UptimeLookbackWindow must be within safe range");
     require(
       window <= getEpochSize().sub(2),
-      "UptimeLookbackWindow must be smaller or equal to epochSize-2"
+      "UptimeLookbackWindow must be smaller or equal to epochSize - 2"
     );
 
-    uptimeLookbackWindow.oldValue = getUptimeLookbackWindow();
+    uptimeLookbackWindow.oldValue = _getUptimeLookbackWindow();
+
+    // changes only take place on the next epoch
+    uptimeLookbackWindow.nextValueActivationEpoch = getEpochNumber().add(1);
     uptimeLookbackWindow.nextValue = window;
 
-    // epoch 0 is the genesis block, thus implies it has not been initialized
-    if (uptimeLookbackWindow.nextValueActivationEpoch == 0) {
-      // on initialize we set the value for the current epoch which configurer
-      // must make it so it's SAME as geth's chainConfig value
-      uptimeLookbackWindow.nextValueActivationEpoch = getEpochNumber();
-    } else {
-      // changes only take place on the next epoch
-      uptimeLookbackWindow.nextValueActivationEpoch = getEpochNumber().add(1);
-    }
     emit UptimeLookbackWindowSet(window, uptimeLookbackWindow.nextValueActivationEpoch);
   }
 
   /**
    * @notice Gets the uptime lookback window.
```

Comment:
Call out that this reverts if there is no lookback window in effect in the current epoch (I don't know if there is a special way to do that or if it's just mentioning that in the @notice)